About.html

```diff
@@ -0,0 +1,137 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <script src="https://code.jquery.com/jquery-3.6.0.min.js"></script>  
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@400;700;900" rel="stylesheet">
+    <title>Document</title>
+</head>
+
+<body>
+
+    <header>
+        <div id="black-header " class="display_desktop"> 
+            <ul class="menu-black">
+               
+                <li><img src="images/facebook.png"></li>
+                <li><img src="images/twitter.png"></li>
+                <li>Logout</li>
+                <li>Mypage</li>
+                <li>Korean</li>
+            </ul>
+            </div>
+        <nav class="display_desktop navigation">
+            
+            <ul class="desktop_menu">
+                <li> <img src="images/logo.png" class="logo"></li>
+                <li>Program</li>
+                <li>Join</li>
+                <li>Sponsor</li>
+                <li>News</li>
+                <li ><div id="CCampgain">CC Campaign</div></li>
```

Comment:
- [ ] [Optional] Well done on adding the button! However, I noticed that the styles of the button do not match the design requirements. To improve the appearance and alignment, I recommend adding some extra padding to the button and reduce the border. This will help it better align with the design and enhance its visual appeal. Please update your code to incorporate the suggested changes. Thank you for your efforts! 🙌

| Your | Professional |
| --- | --- |
| ![image](https://github.com/umarahgul/capstone-1/assets/55840999/8ca6fe80-0bf2-4073-ab7e-03c9bb5bea85) | ![image](https://github.com/umarahgul/capstone-1/assets/55840999/ee1a2696-d3f9-4efb-ae0d-37268b15fe16) |